help.js

```diff
@@ -37,12 +33,13 @@ var keysAndValues = {
 
 var MessageId = {
   type: 'string',
-  test: /^%[a-zA-Z0-9\+\/]+={0,2}\.\w+$/
+  test: /^%[a-zA-Z0-9+/]+={0,2}\.\w+$/
```

Comment:
I removed these escapes because they aren't actually doing anything in this context.